tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+	"compilerOptions": {
+		"module": "umd",
+		"noImplicitAny": true,
+		"removeComments": true,
+		"preserveConstEnums": true,
+		"sourceMap": true,
+		"target": "es5"
```

Comment:
Yes, more or less.

Null and undefined are no longer automatically valid values for every type. Two main effects:

* If you want to make something null/undefined, you have to explicitly say so. You have to make nullable parameters/fields optional (`?`), or use `MyResult | undefined` for variables and function return types.

* If you want to use a maybe undefined/null value as a real value, you have to check it first. That means this stops being ok:

  ```typescript
  var maybe: string | undefined;
  var value: string;
  
  value = maybe; // 'maybe' could be undefined, which isn't valid for 'value'
  ```

  And you have to do this instead:

  ```typescript
  var maybe: string | undefined;
  var value: string;
  
  if (maybe !== undefined) {
    value = maybe; // Type guard narrows the type of 'maybe' here to just 'string'
  }
  ```

Should be easy to drop if it turns out to be annoying - any code that compiles with `strictNullChecks` on also compiles without.